lib/portlayer/event/collector/vsphere/collector.go

```diff
@@ -159,7 +163,21 @@ func evented(ec *EventCollector, page []types.BaseEvent) {
 		return
 	}
 
+	if len(page) == 0 {
+		return
+	}
+
+	// skip events already seen
```

Comment:
Yeah, the two main facets of this PR need to roll into govmomi at some point:
1. option to eliminate previously collected events - can/should we present a simple channel instead of the page callback? (vmware/govmomi#730)
2. re-auth mechanism not tied to keepalive (vmware/govmomi#729)